seq2seq/third_party/utils/sentence_splitter.py

```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
This file looks unmodified other than adding license information. Maybe in this case we should leave it unchanged and only clarify the headers if we have to modify it ourselves later.

Reply:
so you mean to remove license? Jamie once said we need to have a license fle on top of every file, what do you think? thanks

Reply:
I am not 100% sure. I guess this is fine.